packages/core/types/src/analytics/mutations.ts

```diff
@@ -0,0 +1,45 @@
+export interface TrackAnalyticsEventDTO {
+  /**
+   * The event name
+   */
+  event: string
+  /**
+   * The actor of the event, if there is any
+   */
+  actor?: {
+    id: string
+  }
```

Comment:
question/nit: Do we need this to be an object? Could we just do `actorId`?

Reply:
@olivermrbl I did it for two reasons:
1. Make it more flexible (so eg. adding actor-specific data is easier, such as eg. email)
2. Consistency with the group (they are both objects)

We could of course name it `actorId` and then, if we need more data, add an `actor` object later on without causing a breaking change, but I figured the difference is minimal. 

Which one would you prefer? I am happy to go either way

Reply:
I don't have a strong opinion on this, to be honest, but in my experience, the actor/user ID is typically top-level, and the actor's properties or traits are in a separate object. Now that I think of it, this is probably only relevant for the `identify` call since you wouldn't populate actor characteristics on the regular tracking event.

Reply:
@olivermrbl you can populate actor characteristics on the tracking event as well (we just don't currently), which is why I thought having it as an object would make that easier. Just make the final call :D

Reply:
Let's do `actorId` if it's not too much work :)